paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -41,8 +43,12 @@ internal class DefaultLinkAccountManager @Inject constructor(
     private val linkEventsReporter: LinkEventsReporter,
     private val errorReporter: ErrorReporter,
 ) : LinkAccountManager {
+
     override val linkAccount: StateFlow<LinkAccount?> = linkAccountHolder.linkAccount
 
+    private val _paymentDetails: MutableStateFlow<ConsumerPaymentDetails?> = MutableStateFlow(null)
+    override val paymentDetails: StateFlow<ConsumerPaymentDetails?> = _paymentDetails.asStateFlow()
```

Comment:
Alternatively, we could store this on its own repository. Keeping it here facilitates updates when interacting with payment related operations (a follow-up PR will update this on `updatePaymentDetails` calls to keep this screen in sync with the wallet).